### PR TITLE
Actually update Millipede to V04-11-01

### PR DESCRIPTION
**Pull request by iarspider**
Spotted by @mmusich: https://github.com/cms-sw/cmsdist/pull/7660#discussion_r824761342

### CODE REVIEW

**Comment by iarspider** [comment]
please test

File: millepede.spec [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ### RPM external millepede V04-11-01
-%define tag 9ee817fc61fe3e1b6543a8a16f7bcd8e1f8c331f
+%define tag 4f288098f84f274f5a241f921035af77bc07d5c8
```
Comment (smuzaffar) [inline millepede.spec]: @iarspider , in this case please drop the `tag` and just use `%{realversion}` in the `Source:`

Comment (iarspider) [reply millepede.spec]: @smuzaffar like this?

Comment (smuzaffar) [reply millepede.spec]: in this case you might want to update https://github.com/cms-sw/cmsdist/pull/7685/files#diff-19dcda17fe5f30aa2133b7a5fbb298c4471da65a7277dd395a170f1b86deb1b1R6 to match the directory. Just donwload the source , unpack it and see what directory it creates . I guess it mighty create `%{n}-ii-%{realversion}`

Comment (smuzaffar) [reply millepede.spec]: I mean you need to fix `%setup -n %{n}-%{realversion}` statement

**Comment by iarspider** [comment]
please test

**Comment by smuzaffar** [comment]
+externals

**Pull request merged**